Main.java

```diff
@@ -3,58 +3,51 @@
 import java.io.*;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.sql.*;
 import java.util.StringTokenizer;
+import com.company.dbconnection.DatabaseLink;
 
 public class Main {
 
-    static final String FILE_TYPE_TXT = ".txt";
-    static final String FILE_TYPE_JSON = ".json";
-    static final String REMOVE_SPECIAL_CHAR = "[^a-zA-Z0-9 -]";
-    static final String SINGLE_SPACE = "";
-    static String searchResult;
+    static String textFilePath = null;
+    static String searchKeyword = null;
 
     public static void main(String[] args) {
 
-        String textFilePath;
-        String searchKeyword;
-        String errorMessage;
-        if (args.length != 2) {
-            errorMessage = "Provide valid filepath and search keyword";
-            System.out.println(errorMessage);
-            Database.databaseOperation(null, null, searchResult = "Failure", errorMessage, 0);
+        if((args.length != 2) || (args[0] == null || args[0] == " " || args[0] == "") || (args[1] == null || args[1] == " " || args[1] == "")) {

```

Comment:
What is the arguments are 3 empty spaces, what if they are 5 empty spaces? Google and find out how to check if a string is empty, blank or not.

Reply:
Sure. Will do.